include/mbedtls/net.h

```diff
@@ -20,11 +20,11 @@
  *
  *  This file is part of mbed TLS (https://tls.mbed.org)
  *
- * \deprecated Superseded by mbedtls/net_sockets.h
+ * \deprecated Superseded by net_sockets.h
  */
 
 #if !defined(MBEDTLS_DEPRECATED_REMOVED)
-#include "mbedtls/net_sockets.h"
+#include "net_sockets.h"
 #if defined(MBEDTLS_DEPRECATED_WARNING)
 #warning "Deprecated header file: Superseded by mbedtls/net_sockets.h"
```

Comment:
Should we change this one as well?

Reply:
@andresag01 Why do you think it shouldn't change? Although `net.h` is superseded, it can still be included.

Reply:
This was a long time ago, so what I probably meant is that in the message for this warning there is a `mbedtls/net_sockets.h`. Do we want to make that `net_sockets.h` as well? Its a really minor thing and certainly not a stopper, more of a question. I will accept the PR.

Reply:
I think in the message, it's more appropriate to keep it as is, as it is intended for human readers, and it's informative with the current wording

Reply:
I personally think it's better to keep it as it is.